projects/demos/app/demo-modules/kitchen-sink/template.html

```diff
@@ -138,6 +138,11 @@ <h3>
             [(ngModel)]="event.color.secondary"
             (change)="refresh.next()"
           />
+          <input
+            type="color"
+            [(ngModel)]="event.color.secondaryText"
```

Comment:
Please can we make this:

```
[ngModel]="event.color.secondaryText ?? '#1e90ff'"
(ngModelChange)="event.color.secondaryText = $event"
```

So it will default to the blue color used for the text by default?

Reply:
Haven't checked it, but is this necessary? Your color is set via CSS, if Angular's style-color resolves to null it won't to anything, will it?

Reply:
Yeah, further thinking of it, I surely won't do it! Because the style will override the css! And if someone adopted your CSS, he will get your default themes' default color.

Reply:
Yeah sure, just go ahead. This PR is merged already, so just put it in your branch?